src/main/asciidoc/transactions.adoc

```diff
@@ -33,82 +41,257 @@ static class Config extends AbstractCouchbaseConfiguration {
-@Autowired
-CouchbaseClientFactory couchbaseClientFactory;
+@Service
+@Component
+public class PersonService {
+
+    final CouchbaseOperations personOperations;
+
+    public PersonService(CouchbaseOperations ops) {
+        personOperations = ops;
+    }
+
+    @Transactional(transactionManager = BeanNames.COUCHBASE_TRANSACTION_MANAGER) // default, shown for clarity
+    public Person replacePerson(String id) {
+        Person p = personOperations.findById(Person.class).one(id);
+        p.setFirstname(p.getFirstname()+" Junior");
+        return personOperations.replaceById(Person.class).one(p);
+    }
 
-public void doSomething() {
- transactions.run(ctx -> {
-  ctx.insert(couchbaseClientFactory.getDefaultCollection(), "id", "content");
-  ctx.commit();
- });
 }
 ----
 ====
 
-== Object Conversions
+== Transactions with CouchbaseTransactionalOperator
 
-Since the transactions library itself has no knowledge of your spring data entity types, you need to convert it back and
-forth when reading/writing to interact properly. Fortunately, all you need to do is autowire the `MappingCouchbaseConverter` and
-utilize it:
+CouchbaseTransactionalOperator is used for reactive transactions. It is an important class for Couchbase Transactions in Spring Data as there is no Reactive TransactionManager for Couchbase.
+The class simplifies programmatic transaction demarcation and transaction exception handling.
+The central method is transactional(), supporting transactional wrapping of functional sequences code that. This operator handles the transaction lifecycle and possible exceptions such that neither the ReactiveTransactionCallback implementation nor the calling code needs to explicitly handle transactions.
+Typical usage: Allows for writing low-level data access objects that use resources such as database connections but are not transaction-aware themselves. Instead, they can implicitly participate in transactions handled by higher-level application services utilizing this class, making calls to the low-level services via an inner-class callback object.
 
-.Transaction Conversion on Write
+Note that a CouchbaseTransactionalOperator is available as a bean.
+If creating one explicitly, it must be created with the following (and not TransactionalOperator.create(manager)).
+.TransactionalOperator Construction
 ====
 [source,java]
 ----
-@Autowired
-MappingCouchbaseConverter mappingCouchbaseConverter;
+public static CouchbaseTransactionalOperator create(CouchbaseCallbackTransactionManager manager) 
+====
 
-public void doSomething() {
-  transactions.run(ctx -> {
+.Transaction Access Using TransactionalOperator.execute()
+====
+[source,java]
+----
+@Autowired TransactionalOperator txOperator;
 
-   Airline airline = new Airline("demo-airline", "at");
-   CouchbaseDocument target = new CouchbaseDocument();
-   mappingCouchbaseConverter.write(airline, target);
+    Person person = rxCBTmpl.insertById(Person.class).inCollection(cName).one(WalterWhite).block();
```

Comment:
where is the rxCBTmpl defined? the name is not really good for demonstrative purposes I think

Reply:
It's the reactiveCouchbaseTemplate.  I'll give it a better name and show the definition (@Autowired).